contributions/presentation/week3/vemula-akhily/Readme.md

```diff
@@ -0,0 +1,15 @@
+**Topic:Introduction to Chaos Engineering and its tools**
+
+**Subject:Chaos Engineering**
+
+**Description:**
+
+Fault injection vs Chaos engineering differences.
+
+Comparision between chaos machine with other chaos engineering systems (Ex: Simian Army).
+
+Performace evaluation of a spring application using chaos machine/chaos monkey. 
+
+Please note that we will be refering to  "A Chaos Engineering System for Live Analysis and
+Falsification of Exception-handling in the JVM
+" to consider some inputs as we see very little   documentation available.
```

Comment:
Add the following paragraph in the end.

**Relevance and timeliness:**
several sentences to discuss how chaos engineering could be related to DevOps